### DISCUSSION TITLE
Indentation issues

### DISCUSSION THREAD

**Discussion by KotikLina** [TomSchimansky/CustomTkinter#1662] [Q&A] [answered]
It's hard to explain. First I want to mention that the initial display is correct and it is displayed because I removed `self.ToggleFrameResults()` from the code. But if you click on `SwitchForScoring` twice (turn off and on), then the problem will appear, because of which I wrote here.
I do not speak English. I am using a translator.

Объяснить сложно. Для начала хочу упомянуть что изначальное отображение является правильным и оно отображается потому что я убрал `self.ToggleFrameResults()` из кода. А вот если нажать на `SwitchForScoring` дважды(выключить и включить), то появится та проблема, из-за которой я сюда написал.
Я не говорю на английском. Я использую переводчик.
```py
from customtkinter import *


class Main(CTk):
    def __init__(self, focus=False):
        super().__init__()

        self.geometry("500x530")

        self.font1 = CTkFont(size=20, weight="bold")
        self.font2 = CTkFont(size=18)
        self.font3 = CTkFont(size=14)

        self.FrameScoring = CTkFrame(master=self, fg_color="#404040")
        self.FrameScoring.pack(pady=10)

        self.FrameSwitchForScoring = CTkFrame(master=self.FrameScoring)
        self.FrameSwitchForScoring.pack(pady=10)

        self.SwitchForScoring = CTkSwitch(master=self.FrameSwitchForScoring, text="Отобразить подсчёт очков",
                                          font=self.font3, width=348)
        self.SwitchForScoring.pack(padx=10, pady=10)
        self.SwitchForScoring.bind('<Button-1>', lambda event: self.ToggleFrameResults())

        self.FrameResults = CTkFrame(master=self.FrameScoring)
        self.FrameResults.pack(pady=10, padx=15)

    def ToggleFrameResults(self):
        if self.SwitchForScoring.get():
            self.FrameResults.pack()
            self.geometry("500x630")
        else:
            self.FrameResults.pack_forget()
            self.geometry("500x530")


if __name__ == "__main__":
    app = Main()
    app.mainloop()
```
![image](https://github.com/TomSchimansky/CustomTkinter/assets/105449751/b65c0d43-e8da-4f81-ae6f-851a8010b407)
![image](https://github.com/TomSchimansky/CustomTkinter/assets/105449751/d186c7a0-e657-4bac-a695-a11bb0661dd9)

**Comment by KotikLina** [comment]
It's important to note that I'm not changing FrameResults in any way, only changing whether it's displayed or not. If my code is written somehow wrong or it can be improved - write, I will read it (I'm not talking about the error itself, but about the code)

**Comment by LoKiToon** [comment, marked as answer]
On line 30, you have to specify the pack parameters again.
Try adding this to line 30. It should solve the issue. 
![image](https://github.com/TomSchimansky/CustomTkinter/assets/98922449/023ba2dd-c162-4755-8284-b814e919710c)

**Reply by KotikLina** [reply]
Thanks, I did not understand how this frame appears and disappears, but after you wrote this, I seem to understand

**Comment by KotikLina** [comment]
If something needs to be done about this discussion, speak up. I created it for the first time and I may not know how and what to do
